sitemap.xml

```diff
@@ -0,0 +1,184 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset
+        xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9
+            http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd">
+
+    <url>
+        <loc>https://app.electricitymap.org/</loc>
+        <lastmod>2021-10-17T17:23:48+00:00</lastmod>
+    </url>
+
+    <url>
+        <loc>https://app.electricitymap.org/map</loc>
+        <lastmod>2021-10-17T17:23:48+00:00</lastmod>
+    </url>
+
+    <url>
+        <loc>https://app.electricitymap.org/map?lang=ar</loc>
+        <lastmod>2021-10-17T17:23:48+00:00</lastmod>
+    </url>
```

Comment:
Instead of having one entry per language, maybe this solution is better? 

```xml
<url>
        <loc>https://app.electricitymap.org/map</loc>
	<xhtml:link rel="alternate" hreflang="ar" href="https://app.electricitymap.org/map?lang=ar"/>
	<xhtml:link rel="alternate" hreflang="fr" href="https://app.electricitymap.org/map?lang=fr"/>
	...
</url>
```

https://webmasters.stackexchange.com/questions/104784/should-i-put-my-locale-query-param-in-sitemap-url

I don't know what's best though, so if you have any knowledge about this I'd love some inputs!